libpod/runtime_pod_linux.go

```diff
@@ -27,6 +27,12 @@ func (r *Runtime) NewPod(options ...PodCreateOption) (*Pod, error) {
 		return nil, errors.Wrapf(err, "error creating pod")
 	}
 
+	// Set default namespace to runtime's namespace
+	// Do so before options run so they can override it
+	if r.config.Namespace != "" {
```

Comment:
Minor nit: I think we can set it unconditionally and avoid the string compare.

Reply:
Maybe I am not seeing it, but where does the SetNamespace() happen for a given container?

Reply:
I managed to lose that commit in the wash. Should be back now...